configure.ac

```diff
@@ -1062,6 +1062,24 @@ AS_CASE([$ac_sys_system/$ac_sys_release],
  ]
 )
 
+AC_MSG_CHECKING([for --with-emscripten-target])
+AC_ARG_WITH([emscripten-target],
+  [AS_HELP_STRING([--with-emscripten-target=@<:@browser|node@:>@], [Emscripten platform])],
+[
+	AS_VAR_IF([ac_sys_system], [Emscripten], [
+    AS_CASE([$with_emscripten_target],
+      [browser], [ac_sys_emscripten_target=browser],
+      [node], [ac_sys_emscripten_target=node],
+      [AC_MSG_ERROR([Invalid argument: --with-emscripten-target=browser|node])]
```

Comment:
I seem to be able to run configure without passing this at all, but we should probably check that one of these is set if we are targeting `wasm32-unknown-emscripten`

Reply:
The option defaults for ``browser``:
```
AS_VAR_IF([ac_sys_system], [Emscripten], [ac_sys_emscripten_target=browser])
```

Reply:
Ah, still getting the hang of reading autoconf, thanks!